spec/GameSpec.js

```diff
@@ -0,0 +1,36 @@
+describe("Game", function() {
+  var game;
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it('should have a property player1', function() {
+    expect(game.player1).not.toBeUndefined();
+  });
+
+  it('should have a property player2', function() {
+    expect(game.player2).not.toBeUndefined();
+  });
+
+  it('should have a property gameBoard', function() {
+    expect(game.gameBoard).not.toBeUndefined();
```

Comment:
let the game keep control of its properties. Direct access to object properties is usually not good practice. use getters and setters. game.gameBoard becomes game.getGameBoard() which returns game.gameBoard.